internal/util/pkgutil/pkgutil.go

```diff
@@ -21,10 +21,52 @@ import (
 	"strings"
 
 	"github.com/GoogleContainerTools/kpt/pkg/kptfile"
+	"github.com/GoogleContainerTools/kpt/pkg/kptfile/kptfileutil"
 	"sigs.k8s.io/kustomize/kyaml/copyutil"
 )
 
-// walkPackage walks the package defined at src and provides a callback for
+type RemoteSubPkgInfo struct {
+	PackagePath          string
```

Comment:
Can we start using the types for the package path:

https://github.com/GoogleContainerTools/kpt/blob/next/internal/pkg/pkg.go

We want strong typing and compiler help instead of having this as a string everywhere and just using field/variable names to give semantics.

Reply:
Yeah, we can do that. But let me do that in a separate PR as this will create lots of changes that will make this PR much harder to read.